products/jbrowse-web/src/ShareButton.tsx

```diff
@@ -224,7 +224,7 @@ const ShareDialog = observer(
       return () => {
         cancelled = true
       }
-    }, [currentSetting, url, snap])
+    }, [currentSetting, url, snap, session])
```

Comment:
Why is `session` added to the useEffect deps here?

Reply:
good catch, that was left in when I was still doing cleaning of custom callbacks in the session, i've removed it